go.sum

```diff
@@ -20,6 +20,8 @@ github.com/elastic/package-registry v0.6.0 h1:rLdhOAJza3HERy63VZNeHPHsrRyGcEeiBP
 github.com/elastic/package-registry v0.6.0/go.mod h1:ERTTIxAsQOCVZJDqR4LJbDDAtxV+pz4wdPPrKheiAUc=
 github.com/elastic/package-registry v0.7.1 h1:XuWaNMPk+BFLhgrxyQtpkY6yFuJ1Dd2Ms7ltPitUTzo=
 github.com/elastic/package-registry v0.7.1/go.mod h1:oQx3Tg9ynuC6APd0o0OHud9kyPX6S6IzdJp/R4Hj1HY=
+github.com/elastic/package-registry v0.9.0 h1:bfVF5k6w43zjYaF+0t3fRCU1IguRhwqeY1wjG1LMA6k=
+github.com/elastic/package-registry v0.9.0/go.mod h1:oQx3Tg9ynuC6APd0o0OHud9kyPX6S6IzdJp/R4Hj1HY=
```

Comment:
Nit: run `go mod tidy` to clean up older versions of dependencies in this file?

Reply:
Yes, you're right. Thanks!